index.html

```diff
@@ -3,9 +3,48 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel = "stylesheet" type = "text/css" href = "style/index.css" />
+    <title>Liz Shulman Portfolio</title>
 </head>
 <body>
-    
+    <header>
+        <h1>Liz Shulman</h1>
+        <nav>
+            <a class="current" href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="contact.html">Contact</a>
+            <a href="projects.html">Projects</a>
+        </nav>
+    </header>
+    <section class="intro">
+        <div class="left">
+            <img src="https://picsum.photos/800/600" alt="random image">
+        </div>
+        <div class="right">
+            <h2>Liz Shulman</h2>
+            <form>
+                <button type="submit" formaction="projects.html">My Projects</button>
+            </form>
+            <p>
+                Here are some words about me and my journey in coding so far. These words are authentic and inspiring. These words tell my story.
+            </p>
+        </div>
+    </section>
+    <section class="gallery">
```

Comment:
your gallery looks great! this is an awesome way to structure it to be able to display flex on it !